wiki/en/Running-a-Server.md

```diff
@@ -298,6 +297,32 @@ You yourself should connect using the local network (LAN) address of the machine
 
 Most domestic internet connections change their external IP address after a short period. To avoid problems with this, you might want to set up "dynamic DNS" to get a static (sub-)domain you can share with others. Please research how to do that for your specic set up. Your router might support some "dynamic DNS" providers out of the box. If this is not the case, set up a dynamic DNS client as described by the dynamic DNS provider you chose.
 
+## DNS SRV record support
+
+Jamulus client 3.10.0 onwards supports [DNS SRV records ("service" records)](https://en.wikipedia.org/wiki/SRV_record). If Jamulus client finds a SRV record associated with the domain name given in the Connect window, it will try to connect to the endpoint host and port given in the SRV record. It's similar to a HTTP redirect in a web browser, only it's between Jamulus client and the DNS server. Jamulus server is not involved in this transaction, other than listening for connections on the same port given in the SRV record.
```

Comment:
Hmm, actually it was me who suggested adding the version. We do not provide docs for older versions, do we?

For most features it's quite obvious if the docs match the installed version (e.g. the GUI shows the feature or a CLI flag works or doesn't). For this feature, the only sane way for checking availability is the version.

I was mainly hoping to avoid "this doesn't work" support threads.

Reply:
My view is "We only support the current release version".  So if anyone tries "It doesn't work" without stating their version, the first question is "What version?".  Until the Jamulus version supporting the feature is released, the updated docs aren't on "release", they should be on "next-release" - for that reason.